src/sdk-reference/js/6/kuzzle/introduction/index.md

```diff
@@ -10,6 +10,8 @@ order: 0
 The Kuzzle class is the main class of the SDK.  
```

Comment:
I think it's better to state the inheritance list at the very beginning of the description.

Something like this:

```suggestion
Inherits from: [KuzzleEventEmitter]({{ site_base_path }}sdk-reference/js/6/kuzzle-event-emitter).

The Kuzzle class is the main class of the SDK.  
```